src/components/Facility/FacilityCard.tsx

```diff
@@ -226,22 +226,28 @@
                         id="facility-notify"
                         ghost
                         border
-                        className="h-[38px]"
+                        className="h-[38px] tooltip"
                         onClick={(_) => setNotifyModalFor(facility.id)}
                       >
                         <CareIcon icon="l-megaphone" className="text-lg" />
                         <span className="hidden md:block">Notify</span>
+                        <span className="tooltip-text tooltip-top md:hidden">
+                          Notify
+                        </span>
                       </ButtonV2>
                     )}
                     <ButtonV2
                       href={`/facility/${facility.id}`}
                       id="facility-details"
                       border
                       ghost
-                      className="h-[38px]"
+                      className="tooltip h-[38px]"
                     >
                       <CareIcon icon="l-hospital" className="text-lg" />
-                      <span className="hidden md:inline">
+                      <span className="hidden sm:inline">
+                        {t("view_faciliy")}
+                      </span>
+                      <span className="tooltip-text tooltip-top sm:hidden">
                         {t("view_faciliy")}
```

Comment:
Go ahead and do this as well (do the same for other buttons in the page).

We are in the process on moving over to Shadcn, can you replace the ButtonV2s on the page with Button from `components/ui/button`. Use `navigate` for handling onClick and `outline-primary` style variant for now (this might change/require tweaking depending on feedback from others).